manager/config.py

```diff
@@ -0,0 +1,104 @@
+from abc import ABCMeta, abstractmethod
+from genericpath import exists
+from os.path import dirname
+from shutil import copy
+
+from manager.command import EditCommand, SshCommand, AbstractCommand
+
+
+class Window:
+    x = 0
+    y = 0
+    width = 0
+    height = 0
+
+    def __init__(self, x: int, y: int, width: int, height: int):
+        self.x = x
+        self.y = y
+        self.width = width
+        self.height = height
+        pass
+
+
+class Command:
+    _commands = {}
+
+    def add_command(self, name: str, handler: AbstractCommand):
+        self._commands[name] = handler
+
+    def get_command(self, name: str) -> AbstractCommand:
+        return self._commands[name]
+
+
+class Connection:
+    DEFAULT_PORT = 22
+    label = ''
+    host = ''
+    port = 0
+    args = []
+
+    def __init__(self, label: str, host: str, port: int, args: list):
+        self.label = label
+        self.host = host
+        self.port = port
+        self.args = args
+
+
+class Config:
+    _connections = []
+    _window = None
+    _commands = {}
+
+    def add_connection(self, connection: Connection):
```

Comment:
You don't need and getters or setters. Just store it in property, like:
```
class MyClass:
  def __init__(self, connections, commands, *args, **kwargs):
    self.connections = connections
    self.commands = commands
...
my = MyClass()
# set connections
my.connections = connections
# add connection
my.connections.append(some_connection)
# get connection
var = my.connections[0]
# set commands
my.commands = commands
# add command
my.commands['delete_all'] = some_command_to_delete_all
# get command
cmd = my.commands['edit']
```
You get only 1 line instead 4 (blank line, 2 lines for define method, 1 line for calling method)